third_party/open-policy-agent/gatekeeper/helmify/static/values.yaml

```diff
@@ -1,7 +1,86 @@
-controllerManager:
+runtimeConfig:
+  health: {}
+    # healthProbeBindAddress: :8081
+  metrics: {}
+    # bindAddress: 127.0.0.1:8080
+  webhook: {}
+    # port: 9443
+  leaderElection: {}
+    # leaderElect: true
+    # resourceName: e29e094a.k8s.io
+  manager:
+    runtime: containerd
+    otlpEndpoint: ""
+    logLevel: info
+    scheduling: {}
+      # repeatInterval: "" # to be parsed into time.Duration
+      # beginImmediately: true
+    profile: {}
+      # enable: false
+      # port: 0
+    imageJob:
+      successRatio: 1.0 # float; ok with YAML?
+      cleanup: {}
+        # delayOnSuccess: ""
+        # delayOnFailure: ""
+    pullSecrets: [] # image pull secrets for collector/scanner/eraser
+    nodeFilter: {}
+      # type: exclude # must be either exclude|include
+      # selectors:
+      #   - eraser.sh/cleanup.filter
+  components:
+    collector:
+      enable: false
+      image: {}
+        # repo: ""
+        # tag: ""
+      request: {}
+        # mem: ""
+        # cpu: ""
+      limit: {}
+        # mem: ""
+        # cpu: ""
+    scanner:
+      enable: false
+      image: {}
+        # repo: ""
+        # tag: ""
+      request: {}
+        # mem: ""
+        # cpu: ""
+      limit: {}
+        # mem: ""
+        # cpu: ""
+      config: "" # |
+        # cacheDir: /var/lib/trivy
+        # dbRepo: ghcr.io/aquasecurity/trivy-db
+        # deleteFailedImages: true
+        # vulnerabilities:
+        #   ignoreUnfixed: true
+        #   types:
+        #     - os
+        #     - library
+        #   securityChecks: # need to be documented; determined by trivy, not us
```

Comment:
i am not sure what this comment means

Reply:
- [ ] TODO(@pmengelbert) Update documentation.

Reply:
removed comment. documentation should be separate PR

Reply:
see issue #590